src/main/webapp/components/workspaces/workspaces.js

```diff
@@ -417,6 +466,7 @@ export default () => {
   const { refetch, loading, error, data } = useQuery(workspaces)
   const [create] = useCreate()
   const [_delete] = useDelete()
+  const [message, setMessage] = React.useState(null)
```

Comment:
Make sure to remove the useState in the Workspaces component https://github.com/connexta/revelio/blob/c99a7d24b76621986724c860c2a4480a84d07aae/src/main/webapp/components/workspaces/workspaces.js#L236